### ISSUE TITLE
Audio delay 

### ISSUE THREAD

**Issue by klgr** [yuliskov/SmartYouTubeTV#243]
Hi, I'm having problems with audio delay on Sony Bravia XF8305, Android 7. Another small bug - when video is finished the player doesn't go back to the main screen but kinda lingers on a black screen loading. I have to click back button to go to main screen. Automatically play next video is disabled in the menu.
Otherwise great work so far. I prefer your player to the original one

**Comment by yuliskov** [comment]
Can you confirm that problem still exists on the latest version?
https://github.com/yuliskov/SmartYouTubeTV/releases/download/6.16.54/SYTV_Orig_v6.16.54_r.apk

**Comment by klgr** [comment]
Yes there is still a small delay, less obvious than the previous version but still there. I'm using the latest 6.16.54. I tried the Pro alt version and I think I'll stick with it for now, it feels snappier overall and audio sync is almost there.

**Comment by yuliskov** [comment]
@klgr thanks for the reply. I'll try to find what could cause this bug. I've noticed it too. It's device specific and very annoying.

**Comment by Tweety63** [comment]
Hi, I notice that I have a small sound delay too on my TV, it's a Philips One (55PUS7304). Don't have the other bug with the black screen.
